meson.build

```diff
@@ -87,6 +87,14 @@ if get_option('win32') != 'false'
   use_win32 = dependency('libva-win32', required: require_win32).found()
   if use_win32
     libva_utils_flags = [ '-DHAVE_VA_WIN32=1', '-DNOMINMAX']
+    dep_dxheaders = dependency('directxheaders-', required : false)
```

Comment:
> 'directxheaders-'

Strange name. Likely my ignorance of meson. Can you educate me what this means here? Is it matching some group as a wildcard?

Reply:
Oh, this was a typo, thanks for catching it. It should be `directx-headers`.